mtprotosender.py:228 #INFO     [2025-09-26 08:45:58,749] - telethon.network.mtprotosender - 0| Connecting to 149.154.167.51:443/TcpFull...
mtprotosender.py:279 #INFO     [2025-09-26 08:45:58,787] - telethon.network.mtprotosender - 0| Connection to 149.154.167.51:443/TcpFull complete!
handlers.py:168 #INFO     [2025-09-26 08:46:03,452] - root - 0| Handler called. MessageDeleted
mtprotosender.py:319 #INFO     [2025-09-26 08:52:19,270] - telethon.network.mtprotosender - 0| Disconnecting from 149.154.167.51:443/TcpFull...
mtprotosender.py:339 #INFO     [2025-09-26 08:52:19,270] - telethon.network.mtprotosender - 0| Disconnection from 149.154.167.51:443/TcpFull complete!
base_events.py:1758 #ERROR    [2025-09-26 08:52:19,308] - asyncio - 0| Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x00000258A848A5C0>
